### PR TITLE
Moved mqtt classes to the mqtt folder

### PR DESCRIPTION
**Pull request by timherreijgers**

### CODE REVIEW

File: include/mqtt/mqtt_enc28j60_ethernet.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#ifndef __MQTT_ETH_H__
+#define __MQTT_ETH_H__
+
+#include <EthernetENC.h>
+
+#include <inttypes.h>
+
+#include "mqtt.h"
```
Comment (Martijn-Luijckx) [inline include/mqtt/mqtt_enc28j60_ethernet.h:8]: shouldn't this be mqtt/mqtt.h?

File: include/mqtt/mqtt_esp8266_wifi.h [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#define __MQTT_WIFI_H__
+
+#ifndef DOMOTICS_DEBUG_LOG_ENABLED
+#define _ESPLOGLEVEL_ 0
+#endif
+
+#include <Arduino.h>
+#include <WiFiEspClient.h>
+
+#include "mqtt.h"
```
Comment (Martijn-Luijckx) [inline include/mqtt/mqtt_esp8266_wifi.h:11]: idem

**Review by Martijn-Luijckx — APPROVED** [review]

File: include/mqtt/mqtt_esp8266_wifi.h (hunk 2 above)
Comment (timherreijgers) [reply include/mqtt/mqtt_esp8266_wifi.h:11]: it will link to the mqtt.h in the same folder

File: include/mqtt/mqtt_enc28j60_ethernet.h (hunk 1 above)
Comment (timherreijgers) [reply include/mqtt/mqtt_enc28j60_ethernet.h:8]: it will link to the mqtt.h in the same folder

**Pull request merged**